addons/packages/app-toolkit/0.1.0/test/app-toolkit-test.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+func main() {
+	fmt.Println("STARTING TEST")
```

Comment:
Could a `README.md` be created in the `test` directory that describes how to run/execute the tests?